src/Entity.php

```diff
@@ -329,7 +329,9 @@ public function prepareInputForAdd($input)
     {
         /** @var \DBmysql $DB */
         global $DB;
-
+        if($input['clone']){
+            $input['sons_cache'] = NULL;
+         }
```

Comment:
This should probably be done in `Glpi\Features\Clonable::cleanCloneInput()` to fix all `CommonTreeDropdown` implementations.